src/Yasumi/Holiday.php

```diff
@@ -1,4 +1,6 @@
-<?php declare(strict_types=1);
+<?php
```

Comment:
Please do not change this line as normally a docblock (Copyright) appears after this.

Reply:
Sorry, I didn't noticed that I didn't deactivate the auto format feature of my IDE.